v1.3 Update - camelCase Overhaul

Radiumio opened the pull request:
Beginning the camelCase overhaul which will clean-up the config and improve readability of all songs.

Radiumio committed:
camelCase Overhaul

First introduction of camelCase up to line 151 of the config.cpp, plus all songs up to there.

Radiumio committed:
More changes

Pushed more changes, and have also mostly removed the faction-specific songs that didn't make sense anymore.

Ryan180602 committed:
camelCasing `ACTION` music

Ryan180602 committed:
camelCase for `MISC` and `SAD` music + Name change

Display name and file/class name for `SCP Main Theme` has been changed to `Final Flash of Existence` and `RCS_finalFlashOfExistence` respectively.

Ryan180602 committed:
camelCase `RELAX` music + Remaining Song

One song named `RCS_io.ogg` seems to be sitting unutilised in `RELAX`. Fits a good tone for a relaxing song, however proper display name is unknown.

Ryan180602 commented:
For camelCase of `RELAX`, the filenames are not being picked up by GitHub Desktop on my end. `config.cpp` has been already updated for `RELAX` songs, only the filenames are left.

`STEALTH` filenames are causing issues on my end as well. Neither the `config.cpp` file nor the filenames themselves have been touched for the respective files.

Radiumio committed:
Finished off Config

Moved `Thought Process` to `RELAX` since it was mistakenly put in STEALTH at one point. The remainder of `RELAX` and `STEALTH` have been updated appropriately to reflect camelCase.

Co-Authored-By: Shourya Biswas <sdthecyanwyan@users.noreply.github.com>

Radiumio commented:
Finished off the rest and changed one song around. All this should need now is in-game testing to ensure all the directories are pointing to the right files.

Radiumio committed:
Final additions

Two new songs

Radiumio committed:
Hotfix

Fixed a song to resolve overlapping classes
